src/empire/city.c

```diff
@@ -15,12 +17,11 @@
 
 #include <string.h>
 
-#define MAX_CITIES 41
 #define RESOURCES_TO_TRADER_RATIO 60
 #define LAND_TRADER_DELAY_TICKS 4
 #define SEA_TRADER_DELAY_TICKS 30
 
-static empire_city cities[MAX_CITIES];
+static empire_city cities[EMPIRE_CITY_MAX_CITIES];
```

Comment:
In the future, we should  get rid of the hardcoded  city limit.

Reply:
We’ll slate that for the 2030 xml refactor 🙂